packages/basti/src/cli/config/config-parser.ts

```diff
@@ -24,22 +24,40 @@ export type RdsClusterTargetConfig = z.infer<
   typeof RdsClusterTargetConfigParser
 >;
 
-const ElasticacheClusterTargetConfigParser = z
+const ElasticacheRedisClusterTargetConfigParser = z
   .object({
-    elasticacheCluster: z.string(),
+    elasticacheRedisCluster: z.string(),
   })
   .merge(TargetConfigBaseParser);
-export type ElasticacheClusterTargetConfig = z.infer<
-  typeof ElasticacheClusterTargetConfigParser
+export type ElasticacheRedisClusterTargetConfig = z.infer<
+  typeof ElasticacheRedisClusterTargetConfigParser
 >;
 
-const ElasticacheNodeTargetConfigParser = z
+const ElasticacheRedisNodeTargetConfigParser = z
   .object({
-    elasticacheNode: z.string(),
+    elasticacheRedisNode: z.string(),
```

Comment:
The config must also support the old property names for backward compatibility. It can look something like this. (I'm writing this code right here in GitHub so it might not work. It's more of a pseudocode)

```
const ElasticacheRedisNodeTargetConfigParser = z.union([
    z.object({
      elasticacheRedisNode: z.string(),
    }),
    z.object({
      elasticacheNode: z.string(),
    })
  ])
  .merge(TargetConfigBaseParser)
  .transform(config => ({
    elasticacheRedisNode: config.elasticacheRedisNode ?? config.elasticacheNode
  }));
```

The idea it to allow using both the old and the new name but normalize it after to the new name with the `transform` method.